.changeset/lucky-singers-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend': major
```

Comment:
Ooh this definitely has to be moved down to just `patch`, especially now that it's just an internal refactor. Major would be a huge deal and we aren't ready for a proper major bump yet for a long time. :)

Reply:
updated to patch